### PR TITLE
Fix #10: Fixes matching files from their owner packages

### LINKED ISSUE #10: Package tests enlists incorrect ad hoc libraries and binaries
Package tests generates a file `Adhoc_files.json` and its purpose is to enlist the files present in system which are not installed via any RPM packages.
Bug: Even if a library or binary present is installed via  RPM package, package tests is showing it as ad hoc file.

For example: `/usr/bin/sotruss` this file is enlisted in the output of `Adhoc_files.json` if the tooling is ran on the latest `docker.io/centos` image, although this file owned by installed RPM `glibc-common`.

```
➜  ~ docker images docker.io/centos
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
docker.io/centos    latest              50dae1ee8677        17 hours ago        196.7 MB
docker.io/centos    centos7             3635e344e8a0        9 months ago        172.3 MB
➜  ~ docker run -ti docker.io/centos bash
[root@298e3b5fc9cc /]# rpm -qf /usr/bin/sotruss
glibc-common-2.17-106.el7_2.6.x86_64
[root@298e3b5fc9cc /]# rpm -qf /bin/sotruss
glibc-common-2.17-106.el7_2.6.x86_64
[root@298e3b5fc9cc /]# rpm -ql glibc-common | grep /bin/sotruss
/usr/bin/sotruss
[root@298e3b5fc9cc /]# ls -lah /bin/sotruss
-rwxr-xr-x. 1 root root 4.3K May 12 12:47 /bin/sotruss
[root@298e3b5fc9cc /]# ls -lah /usr/bin/sotruss
-rwxr-xr-x. 1 root root 4.3K May 12 12:47 /usr/bin/sotruss
```

### PR DESCRIPTION
**Pull request by navidshaikh**
Fixes #10

### CODE REVIEW

File: Introspection/package_tests.py [hunk 1]
```diff
@@ -155,9 +166,8 @@ def find_adhoc_bins_libs(self, installed_packages=[]):
             # installed by RPM
             print len(bins)
             bins = list(set(bins) - set(self.get_installed_files(package)))
-            print len(bins)
 
-        return bins
+        return [b for b in bins if not self.match_bin_with_pkg(b)]
```
Comment (praveenkumar) [inline Introspection/package_tests.py:170]: Here you want to get metadata only when you have **None** return from match_bin_with_pkg ?

Comment (navidshaikh) [inline Introspection/package_tests.py:170]: No. I want to list down the files present in the system which are not installed via any RPMs. Method `match_bin_with_pkg` uses `rpm` command line utility. If a file is not installed via any RPM, the method returns `None`, I am filtering the files from all the files in the system return None via the mentioned method.

Finding the metadata is job of another method and are different utilities.

Comment (praveenkumar) [inline Introspection/package_tests.py:170]: Ack

**Comment by praveenkumar** [comment]
LGTM

**Comment by navidshaikh** [comment]
Merged.

**Pull request closed without merging**